stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -72,6 +124,125 @@ public View getViewForHighlighting(Object element) {
     return (View)element;
   }
 
+  @Override
+  protected void onGetStyles(View element, Accumulator<StyleInfo> styles) {
+    for (Method method : viewGetters) {
+      boolean caught = false;
+      try {
+        ViewDebug.ExportedProperty annotation = getExportedAnnotation(method);
+        Object returnedValue = method.invoke(element);
+        String cssName = convertGetterNameToCssName(method.getName());
+        String cssValue = String.valueOf(returnedValue);
+        boolean isValueDefault = false;
+
+        // special case id to translate it into a human-readable string
+        if (method.getName().equals("getId")) {
+          cssValue = getIdAttribute(element);
+
+          if (cssValue == null) {
+            cssValue = "(none)";
+          }
+        } else {
+          if (method.getReturnType() == int.class) {
+            int intValue = (int) returnedValue;
+
+            if (canIntBeMappedToString(annotation)) {
+              cssValue += " (" + mapIntToStringUsingAnnotations(intValue, annotation) + ")";
+              isValueDefault = false;
+            } else {
+              isValueDefault |= isNumberDefault(intValue, annotation);
+            }
+          } else if (method.getReturnType() == float.class) {
+            isValueDefault |= isNumberDefault((Number)returnedValue, annotation);
+          }
+        }
+
+        styles.store(new StyleInfo(cssName, cssValue, isValueDefault));
+      } catch (IllegalAccessException e) {
+        caught = true;
```

Comment:
Hmm. This whole approach of trying to use all methods prefixed with `get` and then ignoring the ones that throw exceptions seems to be overreaching and dangerous. We should consider having a stock list of `get` methods that we support instead of vacuuming up *any* that begin with the word `get`.

Also, I'm worried that we'll be spamming logcat with all sorts of warnings that the developer can't do anything about and which are actually our fault.